memcached/memcached.php

```diff
@@ -683,6 +683,7 @@ class Memcached
     public const GET_PRESERVE_ORDER = 1;
 
     /**
+     * (PECL memcached &gt;= 3.0.0)<br/>
```

Comment:
I would rather add it this way:

```suggestion
     * A flag for <b>Memcached::get()</b>, <b>Memcached::getMulti()</b> and
     * <b>Memcached::getMultiByKey()</b> to ensure that the CAS token values are returned as well.<br>
     * Available as of Memcached 3.0.0.
     * @since 3.0.0
     * @link https://php.net/manual/en/memcached.constants.php
     * 
```

You can see it noted similarly for a `HAVE_MSGPACK` const there.